risc0/build/src/lib.rs

```diff
@@ -51,7 +51,7 @@ struct Risc0Method {
 }
 
 impl Risc0Method {
-    fn make_method_id(&self) -> MethodId {
+    fn make_method_id(&self) -> Vec<u8> {
```

Comment:
Why don't we want to keep this as a MethodId type?

Reply:
I think because then I'd need to have another ffi to convert raw bytes into a `MethodId`.

I think all of this is basically temporary; once the pure rust prover lands, then we can unify a bunch of things.